metatron/src/main/java/uk/gov/ida/eidas/metatron/MetatronConfiguration.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.ida.eidas.metatron;
+
+import io.dropwizard.Configuration;
+
+public class MetatronConfiguration extends Configuration {
```

Comment:
This doesn't seem to be referred to anywhere else or imported anywhere else. I guess it's included for future work?

Reply:
It's provided to the `MetatronApplication` class. `MetatronApplication` extends `Application<MetatronConfiguration>`. It's a dropwizard thing. We flesh the config out in the work we're doing at the moment.